webapp/src/css/enketo/medic.less

```diff
@@ -18,10 +18,21 @@
     #form-languages {
       display: none;
     }
-    .option-wrapper {
+
+    .dropdown-menu .option-wrapper {
+      padding: 3px;
       .option-label {
+        width: calc(100% - 34px);
+      }
+    }
+
+    .question:not(.or-appearance-likert):not(.or-appearance-columns) .option-wrapper {
+      padding: 5px;
+      .option-label.active {
         white-space: normal;
-        word-break: normal;
+        word-break: break-all;
+        margin: 0;
+        display: contents;
```

Comment:
I don't think `display: contents` is valid in this context. I get quite a nice result if I remove these two lines altogether...

```suggestion
```

Your PR:

![Screenshot from 2023-10-25 12-19-12](https://github.com/medic/cht-core/assets/2064938/d442efc3-c021-4dda-a126-a57981b9b08e)

My suggestion:

![Screenshot from 2023-10-25 12-19-02](https://github.com/medic/cht-core/assets/2064938/2d7056a9-891f-4d9e-aec0-6a42f3efc0ce)

Reply:
We need the ` display: contents;` so radios and checkboxes utilize the space efficiently and display nicely when in columns. 
This is _without_ ` display: contents;`
<img width="500" alt="Screenshot 2023-10-25 at 11 50 24 am" src="https://github.com/medic/cht-core/assets/66472237/98e7afbd-760a-41a2-92c4-cd749346402d">
This is _with_ ` display: contents;`
<img width="500" alt="Screenshot 2023-10-25 at 11 52 40 am" src="https://github.com/medic/cht-core/assets/66472237/8c26a342-93a9-4632-8bb4-3938ee4cd1fe">

Reply:
I've removed the margin

Reply:
Ahh ok, thanks.